### PR TITLE
AST-3243 - Fix: Pagination jump on mobile devices issue

### PR DESCRIPTION
**Pull request by Sudhanshu-1508**
### Description

[https://brainstormforce.atlassian.net/browse/AST-3243](https://brainstormforce.atlassian.net/browse/AST-3243)
### Screenshots

[BEFORE](https://d.pr/v/lfOqkF)
[AFTER](https://d.pr/v/RSBN3I)
### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/dynamic-css/pagination.php [hunk 1]
```diff
@@ -196,6 +196,16 @@ function astra_pagination_css( $dynamic_css ) {
                     }';
 			}
 		}
+		$css_output_mobile   = array(
+
+			'.ast-pagination .next:focus' => array(
+				'width' => '100% !important',
```
Comment (yssambare12) [inline inc/dynamic-css/pagination.php:202]: @Sudhanshu-1508 Can we avoid user !important?

Comment (Sudhanshu-1508) [reply inc/dynamic-css/pagination.php:202]: @yssambare12  !important applies only in mobile mode for pagination block only it won't affect other functionality.

Comment (imnavanath) [reply inc/dynamic-css/pagination.php:202]: Let's get this tested then. Be in loop with Simran @Sudhanshu-1508

**Review by imnavanath — APPROVED** [review]

File: inc/dynamic-css/pagination.php (hunk 1 above)
Comment (imnavanath) [reply inc/dynamic-css/pagination.php:202]: LGTM!

File: inc/dynamic-css/pagination.php [hunk 2]
```diff
@@ -196,6 +196,16 @@ function astra_pagination_css( $dynamic_css ) {
...
+			),
+			'.ast-pagination .prev:focus' => array(
+				'width' => '100% !important',
+			),
```
Comment (imnavanath) [inline inc/dynamic-css/pagination.php:206]:
```suggestion
			'.ast-pagination .next:focus, .ast-pagination .prev:focus' => array(
				'width' => '100% !important',
			),
```

File: inc/dynamic-css/pagination.php [hunk 3]
```diff
@@ -196,6 +196,13 @@ function astra_pagination_css( $dynamic_css ) {
                     }';
 			}
 		}
+		$css_output_mobile   = array(
+
```
Comment (imnavanath) [inline inc/dynamic-css/pagination.php:200]:
```suggestion
```

**Pull request merged**